INSTALL.md

```diff
@@ -34,7 +34,7 @@ The following steps will download and verify the debian package
 Note that it is important to use `apt` and not `dpkg` here - Apt will download dependencies that the
 package needs.
 
-   sudo apt install ./$(cat ubuntu-20.04.latest.txt)
+   sudo apt install `./$(cat ubuntu-20.04.latest.txt)`
```

Comment:
This is very confusing to the casual reader that browses the sources of the markdown. If you copy/paste this into a shell, it will not do what you want. The issue is the missing/wrong indent, so that needs to be fixed (so the whole line renders as code).